Library/Homebrew/cmd/outdated.rb

```diff
@@ -28,10 +28,18 @@ def outdated
 
   def print_outdated(formulae)
     verbose = ($stdout.tty? || ARGV.verbose?) && !ARGV.flag?("--quiet")
+    fetch_head = ARGV.fetch_head?
 
-    formulae.select(&:outdated?).each do |f|
+    formulae.select do |f|
+      f.outdated?(:fetch_head => fetch_head)
+    end.each do |f|
```

Comment:
I have to admit that I'm not a big fan of chaining multiple multi-line `do` blocks. Would the following be also acceptable to you?

```ruby
outdated_formulae = formulae.select do |f|
  f.outdated?(:fetch_head => fetch_head)
end

outdated_formulae.each do |f|
  …
```